config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -0,0 +1,47 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-azure:
+  - name: pull-cluster-api-provider-azure-build
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-azure
+    labels:
+      preset-service-account: "true"
```

Comment:
The service account isn't necessary for decorate: true jobs unless they are creating GCP resources (EG GCE clusters)

Reply:
Removed the unnecessary label :)